src/main/kotlin/frc/robot/subsystems/swerve/SwerveDrive.kt

```diff
@@ -346,7 +369,11 @@ class SwerveDrive private constructor
         Arrays.stream(modules).forEach { obj: SwerveModule? -> obj!!.updateInputs() }
         updateGyroInputs()
         updateModulePositions()
-        estimator.update(odometryYaw, modulePositions)
+
+        if (!isSkidding() || isColliding()) {
```

Comment:
I believe it should be that way?
```suggestion
        if (!(isSkidding() || isColliding())) {
```

Reply:
b3d0839af1dd7b8f4234cdc40796a3e1cfe52940

Reply:
U sure? That way, for example, you will update the PE if the robot is just skidding and not also colliding. Don't you want to reject the measurement in that case?

Reply:
a958a6465b4213b020e2a0d0f67196ec409f178a

Reply:
a958a6465b4213b020e2a0d0f67196ec409f178a